### PR TITLE
separate featurize_patients into non-label and label version

### PR DESCRIPTION
**Pull request by woffett**
Signed-off-by: Jonathan Li <jlli@stanford.edu>

### CODE REVIEW

File: ehr_ml/clmbr/__init__.py [hunk 1]
```diff
@@ -372,7 +373,80 @@ def test_helper(is_val: bool, sample_size: int) -> float:
...
+        os.path.join(model_dir, "info.json"),
+        data,
+        data
+    )
+
+    patient_id_to_info = defaultdict(dict)
+    for i, (pid, index) in enumerate(zip(patient_ids, day_offsets)):
+        patient_id_to_info[pid][index] = i
+
+    patient_representations = np.zeros((len(patient_ids), config['size']))
```
Comment (Lalaland) [inline ehr_ml/clmbr/__init__.py:417]: Code like this probably shouldn't use print.

File: ehr_ml/clmbr/__init__.py [hunk 2]
```diff
@@ -372,7 +373,80 @@ def test_helper(is_val: bool, sample_size: int) -> float:
...
+                )
+                for i, patient_id in enumerate(batch["pid"]):
+                    for index, target_id in patient_id_to_info[patient_id].items():
+                        patient_representations[target_id, :] = embeddings[
+                            i, index, :
+                        ]
+
+    return patient_representations
+
+def featurize_patients_w_labels(model_dir: str, extract_dir: str,
```
Comment (Lalaland) [inline ehr_ml/clmbr/__init__.py:448]: This should call featurize_patients to avoid duplicate code.

File: ehr_ml/clmbr/__init__.py [hunk 3]
```diff
@@ -17,6 +17,7 @@
 from pathlib import Path
 from collections import defaultdict
 from shutil import copyfile
+from tqdm import tqdm
```
Comment (Lalaland) [inline ehr_ml/clmbr/__init__.py:20]: We aren't actually using this correctly? I see you initialize but never update the progress bar.

**Review by Lalaland — COMMENTED** [review]
Looks mostly good with a couple of minor changes.

File: ehr_ml/clmbr/__init__.py (hunk 3 above)
Comment (woffett) [reply ehr_ml/clmbr/__init__.py:20]: The progress bar automatically tracks when you start iterating over it, so there's no need to manually update. You can see the final progress bar in the notebooks in piton_private.

File: ehr_ml/clmbr/__init__.py (hunk 1 above)
Comment (woffett) [reply ehr_ml/clmbr/__init__.py:417]: Removed in the latest commit. Any reason for this though? It doesn't seem like this function merits its own logging setup.

File: ehr_ml/clmbr/__init__.py (hunk 2 above)
Comment (woffett) [reply ehr_ml/clmbr/__init__.py:446]: Addressed in the latest commit.

**Pull request merged**